spring-cloud-stream-binder-kinesis-core/src/main/java/org/springframework/cloud/stream/binder/kinesis/properties/KinesisConsumerProperties.java

```diff
@@ -19,8 +19,18 @@
 /**
  * 
  * @author Peter Oates
+ * @author Jacob Severson
  *
  */
 public class KinesisConsumerProperties {
 
+	private int connectTimeout = 60000;
+
+	public int getConnectTimeout() {
+		return connectTimeout;
```

Comment:
I know that `requireThis` rule isn't enabled in the SCSt's `checkstyle.xml`, but would be great to keep it on board.
So, the code here must be:
```
return this.connectTimeout;
```
At least it is much easier to review from here on GH.